splink/linker.py

```diff
@@ -535,6 +535,17 @@ def _initialise_df_concat_with_tf(self, materialise=True):
 
         return nodes_with_tf
 
+    def _enqueue_df_concat_with_tf(self, pipeline: SQLPipeline):
```

Comment:
This is no good because it doesn't deal with the cache

Reply:
in that case, a fresh sql pipeline is passed in.  If it's in the cache, then the pipeline is returned with it as an input dataframe queued up

if it's not in the cache either 
- materialisation case: a pipeline is returned with it in the cache 
- non-materialisation case:  a pipeline is returned with sql enqueued